.github/workflows/test.yml

```diff
@@ -65,7 +65,7 @@ jobs:
             toxpython: 'python3.6'
             python_arch: 'x64'
             tox_env: 'py36-pytest70-xdist250-coverage62'
-            os: 'ubuntu-latest'
+            os: 'ubuntu-20.04'
```

Comment:
The latest LTS is 22-04 ?

Reply:
GitHub Actions: Run Python 3.6 on Ubuntu 20.04
* We have to run [legacy Python versions](https://devguide.python.org/versions/#unsupported-versions) on an older version of Ubuntu because GitHub Actions does not guarantee support of Python versions that the CPython team no longer supports.
* https://github.com/actions/setup-python/issues/561#issuecomment-1339798843
* https://devguide.python.org/versions